packages/frontend/editor-ui/src/components/canvas/experimental/components/ExperimentalEmbeddedNdvMapper.vue

```diff
@@ -49,74 +44,40 @@ defineExpose({
 </script>
 
 <template>
-	<ElPopover
-		:visible="isVisible"
-		placement="left-start"
-		:show-arrow="false"
-		:popper-class="`${$style.component} ignore-key-press-canvas`"
-		:width="360"
-		:offset="8"
-		append-to="body"
-		:popper-options="{
-			modifiers: [
-				{ name: 'flip', enabled: false },
-				{
-					// Ensures that the popover is re-positioned when the reference element is resized
-					name: 'custom modifier',
-					options: {
-						refX: refBounding.x.value,
-						refY: refBounding.y.value,
-						width: contentSize.width.value,
-						height: contentSize?.height.value,
-					},
```

Comment:
Nice that the popover automatically follows anchor's resizing/re-positioning without this